d3d9ex/stdafx.h

```diff
@@ -24,4 +24,6 @@
 #include <array>
 #include <vector>
 #include <map>
-#include <unordered_map>
\ No newline at end of file
+#include <unordered_map>
+
+#include "Context.h"
```

Comment:
including `Context.h` in precompiled headers pulls most other includes including spdlog, much faster compile